dbms/src/Functions/tests/gtest_unix_timestamp.cpp

```diff
@@ -86,17 +86,17 @@ try
         executeFunction(func_name_dec, createConstColumn<MyDateTime>(std::make_tuple(6), 10, date_time_with_fsp_data[0])));
     /// case 2, func(nullable(not null const))
     ASSERT_COLUMN_EQ(
-        createConstColumn<Nullable<UInt64>>(10, date_result[0]),
-        executeFunction(func_name_int, createConstColumn<Nullable<MyDate>>(10, date_data[0])));
+        createConstColumn<UInt64>(10, date_result[0]),
+        executeFunction(func_name_int, createConstColumn<MyDate>(10, date_data[0])));
     ASSERT_COLUMN_EQ(
-        createConstColumn<Nullable<UInt64>>(10, date_time_int_result[0]),
-        executeFunction(func_name_int, createConstColumn<Nullable<MyDateTime>>(std::make_tuple(0), 10, date_time_data[0])));
+        createConstColumn<UInt64>(10, date_time_int_result[0]),
+        executeFunction(func_name_int, createConstColumn<MyDateTime>(std::make_tuple(0), 10, date_time_data[0])));
     ASSERT_COLUMN_EQ(
-        createConstColumn<Nullable<Decimal64>>(std::make_tuple(12, 0), 10, date_time_decimal_result[0]),
-        executeFunction(func_name_dec, createConstColumn<Nullable<MyDateTime>>(std::make_tuple(0), 10, date_time_data[0])));
+        createConstColumn<Decimal64>(std::make_tuple(12, 0), 10, date_time_decimal_result[0]),
+        executeFunction(func_name_dec, createConstColumn<MyDateTime>(std::make_tuple(0), 10, date_time_data[0])));
     ASSERT_COLUMN_EQ(
-        createConstColumn<Nullable<Decimal64>>(std::make_tuple(18, 6), 10, date_time_with_fsp_result[0]),
```

Comment:
Can we convert `Const<Nullable<Type>>` to `Const<Type>` in `FunctionTestUtils`, and then we don't need to modify the unit test.

Reply:
This is an intentional behavior, because in TiDB, it will generate a constant expr with nullable type but not null value, we need to cover this case, if we convert `Const<Nullable<Type>>(not_null_value)` to `Const<Type>(not_null_value)` in `FunctionTestUtils`, we don't have case to cover this.

Reply:
Did we cover this case in the unit test?
what happens when `createConstColumn<Nullable<Decimal64>>(std::make_tuple(18, 6), 10, date_time_with_fsp_result[0])`?

Reply:
Oh, my mistake, at the beginning I just remove all the tests withe `Const<Nullable<Type>>(not_null_value)`, then I realized that I need to keep these tests, because TiDB will geneate a constant expr with nullable type but not null value. So I decide to updated these tests' result instead of just remove them. Unfortunately I forgot to update the tests here.